# Message to command virtual and real arm. Contain pwm for all joints including gripper

int32 waist
int32 shoulder
int32 elbow
int32 wrist_angle
int32 wrist_rotate
int32 gripper